src/redux/userSlice.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.

Reply:
Remover all the disable file linter checks